packages/app/obojobo-repository/server/migrations/20191216172002-create-drafts-metadata.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+var dbm
+var type
+var seed
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+	dbm = options.dbmigrate
+	type = dbm.dataType
+	seed = seedLink
+}
+
+exports.up = function(db) {
+	return db
+		.createTable('drafts_metadata', {
+			draft_id: {
+				type: 'UUID',
+				notNull: true,
+				defaultValue: new String('uuid_generate_v4()')
+			},
+			key: {
+				type: 'varchar',
+				length: 50,
```

Comment:
I think this may also need to be 256 just in case
```suggestion
				length: 256,
```